docs/news.d/devguide-improvments.rst

```diff
@@ -1,7 +1,16 @@
-.. news-prs: 4281
+.. news-prs: 4281 4294 4296 4279 4238 4234 4237 4231 4233 4210
```

Comment:
To be honest, this looks weird in the news as each item is getting all of these PRs and this isn't really how I envisioned this being used. The idea was that each PR would have a news fragment for every set of related new-worthy changes, then subsequent PRs could tweak them. This was to try to mirror the way PRs worked. Do we need a way to declare multiple different PR sets per file as a convenience?

Reply:
I don't think so.  I can break them out as individual fragments.